src/nunit.analyzers/ClassicModelAssertUsage/AreEqualClassicModelAssertUsageCodeFix.cs

```diff
@@ -18,6 +18,15 @@ public sealed class AreEqualClassicModelAssertUsageCodeFix
 
         protected override void UpdateArguments(Diagnostic diagnostic, List<ArgumentSyntax> arguments)
         {
+            // If named parameters are used, we can't assume that the first argument is the expected one and the second is the actual.
+            // Therefore, start by finding the expected argument and the actual argument.
+            var expectedPosition = arguments.FindIndex(arg => arg.NameColon?.Name.Identifier.Text == NUnitFrameworkConstants.NameOfExpectedParameter);
+            var actualPosition = arguments.FindIndex(arg => arg.NameColon?.Name.Identifier.Text == NUnitFrameworkConstants.NameOfActualParameter);
+
+            // If named arguments are not used, the first argument is expected, and the second is actual.
+            var expectedArgument = expectedPosition != -1 ? arguments[expectedPosition] : arguments[0];
+            var actualArgument = actualPosition != -1 ? arguments[actualPosition] : arguments[1];
+
```

Comment:
Instead of first finding the index, this can be combined into:
```
var expectedArgument = arguments.SingleOrDefault(arg => arg.NameColon?.Name.Identifier.Text == NUnitFrameworkConstants.NameOfExpectedParameter) ?? arguments[0];
var actualArgument = arguments.SingleOrDefault(arg => arg.NameColon?.Name.Identifier.Text == NUnitFrameworkConstants.NameOfActualParameter) ?? arguments[1];
```